README.md

```diff
@@ -172,6 +172,33 @@ directory.
 
 This behavior cannot currently be disabled, PR welcome.
 
+## Code Signing
+
+```yaml
+jobs:
+  build:
+    runs-on: macos-latest
+    steps:
+      - use: mxcl/xcodebuild@v1
+        with:
+          code-sign-certificate: ${{ secrets.CERTIFICATE_BASE64 }}
+          code-sign-certificate-passphrase: ${{ secrets.CERTIFICATE_PASSPHRASE}}
+```
+
+> This feature requires macOS.
+
+A code signing certificate can be installed to the macOS Keychain. It is
+automatically removed from the Keychain in a post action.
+
+To export your certificate from Xcode and Base64 encode it, follow
+[these instructions][export]. Store any secrets, including certificates and
+passphrases, in GitHub as [Encrypted Secrets][secrets].
+
+You may specify a `code-sign-identity` to override any `CODE_SIGN_IDENTITY`
+specified by your project.
+
+To disable code signing, you can specify `code-sign-identity: '-'`.
```

Comment:
Should this perhaps be the default to make our action “just work” more?

Reply:
Maybe? It'd be a breaking change (semver). It'd also require extra work for folks who have already installed certificates via some other means (via other actions, or self-hosted runners). I think using `xcodebuild`'s default behavior (defined by the project) is probably best.